Marlin/example_configurations/K8400/readme.md

```diff
@@ -0,0 +1,15 @@
+# Configuration for Velleman K8400 Vertex
```

Comment:
`README.md` (all-caps) is more proper (and we should also fix this for Hephestos).